java/README.md

```diff
@@ -25,13 +25,25 @@ install:
  - java 7 or later
  - maven 3.3 or later
 
-## Building running tests
+## Building and running tests
 
 ```
 cd java
 mvn install
 ```
 
+## Building and running tests for gandiva (optional)
+
+[Gandiva cpp][2] must be built before this step.
+
+```
+cd java
+mvn install -P gandiva -pl gandiva -am
```

Comment:
please add the gandiva cpp directory too..

Reply:
done